go.mod

```diff
@@ -79,6 +79,8 @@ require (
 	cloud.google.com/go/storage v1.28.0 // indirect
 	github.com/Masterminds/goutils v1.1.1 // indirect
 	github.com/Masterminds/semver/v3 v3.2.0 // indirect
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
```

Comment:
This dependency hasnt been updated in 3 years

https://github.com/alecthomas/template

Can we manage without it? Or is it possible to use another more up to date dependency?